### ISSUE TITLE
error

### ISSUE THREAD

**Issue by ghost** [Eloncase/RainbowSixSiege-CSharp-API#1]
"The remote server returned an error: (404) not found."

**Comment by Eloncase** [comment]
Yes, authorization process changed on ubi side. I'll look into into it but not until next week.
